src/components/profile/Profile.js

```diff
@@ -28,6 +29,15 @@ const Profile = () => {
         console.log(err.response);
       });
   }, [params]);
+  
+  useEffect(() => {
```

Comment:
This should be part of the refactor - we should pass cohort name as part of user data, rather than separately.